pkg/sql/catalog/tabledesc/table.go

```diff
@@ -576,3 +576,17 @@ func PrimaryKeyString(desc catalog.TableDescriptor) string {
 	}
 	return f.CloseAndGetString()
 }
+
+const (
+	// LatestPrimaryIndexDescriptorVersion is the latest index descriptor version
+	// value for primary indexes, and so will be found in all newly-created
+	// primary indexes.
+	// This property is tested by TestLatestIndexDescriptorVersionValues.
+	LatestPrimaryIndexDescriptorVersion descpb.IndexDescriptorVersion = descpb.PrimaryIndexWithStoredColumnsVersion
+
+	// LatestNonPrimaryIndexDescriptorVersion is the latest index descriptor
+	// version value for non-primary indexes, and so will be found in all
+	// newly-created secondary indexes, as well as index mutations.
+	// This property is tested by TestLatestIndexDescriptorVersionValues.
+	LatestNonPrimaryIndexDescriptorVersion descpb.IndexDescriptorVersion = descpb.StrictIndexColumnIDGuaranteesVersion
```

Comment:
Can these live here instead: https://github.com/cockroachdb/cockroach/blob/4369b2166fb4041cca49347efc271eaacd50fbad/pkg/sql/catalog/descpb/structured.go#L96-L122

Reply:
I thought it would be weird to have the test covering them in a different package, that's all.